lambda/audit.py

```diff
@@ -0,0 +1,319 @@
+import json
+import os
+import uuid
+
+import boto3
+from botocore.exceptions import ClientError
+
+import github_api
+from logger import LOG
+
+
+def start(message=None):
```

Comment:
Is this function called from anywhere?

Reply:
Yes it's called when you invoke the lambda with `{"action":"audit"}`

Reply:
I've got it now, thanks.